tests/utils/test_dag_processing.py

```diff
@@ -168,6 +169,7 @@ def test_set_file_paths_when_processor_file_path_is_in_new_file_paths(self):
             processor_factory=MagicMock().return_value,
             processor_timeout=timedelta.max,
             signal_conn=MagicMock(),
+            pickle_dags=True,
```

Comment:
We should use False in all these tests, which is I think the default?

Reply:
OK, I'll replace those change with `False`.